modules/installation-configuration-parameters.adoc

```diff
@@ -761,6 +765,14 @@ host must trust the certificate.
 |If you provide the VPC instead of allowing the installation program to create the VPC for you, specify the subnet for the cluster to use. The subnet must be part of the same `machineNetwork[].cidr` ranges that you specify. For a standard cluster, specify a public and a private subnet for each availability zone. For a private cluster, specify a private subnet for each availability zone.
 |Valid subnet IDs.
 
+|`compute.platform.aws.rootVolume.kmsKeyARN`
+|The Amazon Resource Name (key ARN)) of a KMS key. This is required to encrypt cluster worker nodes OS volumes with specific KMS key.
+|Valid link:https://docs.aws.amazon.com/kms/latest/developerguide/find-cmk-id-arn.html[key ID or the key ARN].
+
+|`controlPlane.platform.aws.rootVolume.kmsKeyARN`
+|The key ARN of a KMS key. This is required to encrypt cluster master nodes OS volumes with specific KMS key.
```

Comment:
> The Amazon Resource Name (key ARN)) of a KMS key.
> The key ARN of a KMS key.

it's better keep the text consistent for master and worker node

Reply:
Addressed